### PR TITLE
Update Kubevirt configuration required by KKP

### PR DESCRIPTION
**Pull request by hdurand0710**
ref kubermatic/kubermatic#8493
Signed-off-by: Helene Durand <helene@kubermatic.com>

### CODE REVIEW

File: content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md [hunk 1]
```diff
@@ -21,6 +21,32 @@ KKP supports KubeVirt Operator >= 0.19.0 and the Containerized Data Importer >=
 to run KubeVirt, however a Kubernetes cluster consists of 3 nodes with 2 CPUs, 4GB of RAM and 30GB of storage, to have a
 minimal installation.
 
+### KubeVirt Configuration Requirements
+Kubevirt requires the following configuration to be used with KKP.
+- In case your Kubevirt namespace has the ConfigMap 'kubevirt-config' then use this ComfigMap for adding the feature gates to it. Look at the path `{.data.feature-gates}`
```
Comment (sankalp-r) [inline content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md:26]: Small typo, instead `Comfigmap` it should be `ConfigMap`

Comment (hdurand0710) [reply content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md]: Fixed.

File: content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md [hunk 2]
```diff
@@ -21,6 +21,32 @@ KKP supports KubeVirt Operator >= 0.19.0 and the Containerized Data Importer >=
 to run KubeVirt, however a Kubernetes cluster consists of 3 nodes with 2 CPUs, 4GB of RAM and 30GB of storage, to have a
 minimal installation.
 
+### KubeVirt Configuration Requirements
+Kubevirt requires the following configuration to be used with KKP.
```
Comment (themue) [inline content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md:25]: Not a lot but please take care for how it's written, so change here and below to `KubeVirt`.

File: content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md [hunk 3]
```diff
@@ -21,6 +21,32 @@ KKP supports KubeVirt Operator >= 0.19.0 and the Containerized Data Importer >=
...
+- In case your Kubevirt namespace has the ConfigMap 'kubevirt-config' then use this ConfigMap for adding the feature gates to it. Look at the path `{.data.feature-gates}`
+- Otherwise, add the feature gate to the resource of type Kubevirt. There should be a single resource of this type and its name is irrelevant
```
Comment (themue) [inline content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md:27]: Hmm, sounds a bit brusque. Maybe better

`Only one resource of this type may be here and its name can be chosen arbitrarily.`

Also take care for interpunction.

File: content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md [hunk 4]
```diff
@@ -21,6 +21,32 @@ KKP supports KubeVirt Operator >= 0.19.0 and the Containerized Data Importer >=
...
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+metadata:
+  name: kubevirt
+  namespace: kubevirt
+spec:
+  configuration:
+    developerConfiguration:
+      featureGates:
```
Comment (themue) [inline content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md:39]: Is the meaning of these feature gates obvious to everyone concerned or does it make sense to add a description to the documentation?

File: content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md (hunk 2 above)
Comment (hdurand0710) [reply content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md]: Fixed everywhere. Thanks

File: content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md (hunk 3 above)
Comment (hdurand0710) [reply content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md]: Updated.

File: content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md (hunk 4 above)
Comment (hdurand0710) [reply content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md:39]: I've added a link the the KubeVirt documentation.

Comment (themue) [reply content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md:39]: Top, thanks.

**Review by themue — APPROVED** [review]
/lgtm
/approve

**Pull request merged**